### PR TITLE
Fix run tests

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

**Comment by pakrym** [comment]
@dotnet-bot test Windows_NT Debug Build please
@dotnet-bot test Windows_NT Release Build please

File: test/dotnet-run.Tests/project.json [hunk 1]
```diff
@@ -19,5 +19,9 @@
     }
   },
 
+  "content": [
+    "../../TestAssets/TestProjects/RunTestsApps/**/*"
```
Comment (piotrpMSFT) [inline test/dotnet-run.Tests/project.json]: Why is this necessary?

@Sridhar-MS

Comment (pakrym) [inline test/dotnet-run.Tests/project.json]: Why shouldn't it be? Test still come from AppBase  which is dll location

Comment (pakrym) [inline test/dotnet-run.Tests/project.json]: I'm wrong, it uses RepoRoot, i'll fix the PR.

**Comment by pakrym** [comment]
@dotnet-bot test this please

**Comment by pakrym** [comment]
@dotnet-bot test this please
